docs/user/openstack/README.md

```diff
@@ -48,7 +48,7 @@ enough to store the ignition config files, so they are served by swift instead.
 `openstack image create --container-format=bare --disk-format=qcow2 --file rhcos-${RHCOSVERSION}-openstack.qcow2 rhcos-${RHCOSVERSION}`
 
 **NOTE:** Depending on your OpenStack environment you can upload the RHCOS image
-as `raw` or `qcow2`. See [Disk and container formats for images](https://docs.openstack.org/image-guide/image-formats.html) for more information.
+as `raw` or `qcow2`. See [Disk and container formats for images](https://docs.openstack.org/image-guide/image-formats.html) for more information. And the imaeg by default must use `rhcos` as prefix unless you overwrite environment variable `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE` (which is not a recommanded way).
```

Comment:
The typo `imaeg` vs `image` is still here.

In my understanding, the image has to be named exactly `rhcos`. Perhaps a better wording could be:

> The installer looks for an image named `rhcos`. This could be overridden via the `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE` environment variable if for instance your cloud operator provides the image under a different name.

Reply:
my bad, I will update accordingly , thanks for the comments and guides